client/components/tinymce/plugins/simple-payments/plugin.jsx

```diff
@@ -9,16 +9,15 @@ import { unmountComponentAtNode } from 'react-dom';
  * Internal Dependencies
  */
 import SimplePaymentsDialog from './dialog';
+import { serialize } from './shortcode-utils';
 import { renderWithReduxStore } from 'lib/react-helpers';
 
 const simplePayments = editor => {
 	let node;
 	const store = editor.getParam( 'redux_store' );
 
 	editor.on( 'init', () => {
-		node = editor.getContainer().appendChild(
-			document.createElement( 'div' )
-		);
+		node = editor.getContainer().appendChild( document.createElement( 'div' ) );
```

Comment:
I noticed in your PRs that you sometimes update/change the code formatting. I think the previous code formatting was valid and nice -- can we please not change it unnecessarily? :) It produces bigger diffs and is not really needed.